source/_integrations/zwave_js.markdown

```diff
@@ -640,7 +630,48 @@ Node {{ node }};{{ s.name }};{{ s.entity_id }}{% endfor %}
 
 {% endraw %}
 
-### How can I add (include) a new device to my Z-Wave network?
+#### Can I switch between the Official Z-Wave JS add-on and Zwavejs2Mqtt?
+
+You can, but you cannot run them both at the same time. Only one of them can be active at the same time.
+
+#### How do I switch between the Official Z-Wave JS add-on and Zwavejs2Mqtt?
+
+Switching does not require renaming your devices.
+
+1. Note your network security keys from your current add-on, then uninstall that add-on. **Do not remove the Z-Wave JS integration or you will lose all device and entity naming.**
```

Comment:
I don't think it reinstalls that quickly. We tell people my steps all the time on discord and they do it. Wouldn't it be easier to have them reconfigure to uncheck the box then to disable it?

Reply:
Yes, they can check the config as long as the add-on isn't uninstalled.

Reply:
> Wouldn't it be easier to have them reconfigure to uncheck the box then to disable it?

What do you mean?

Reply:
I mean is it better for them to disable it, or to do reconfigure it and uncheck the box at step 1.

Reply:
Currently just reconfiguring and unchecking the box to use the official add-on won't disconnect from the server and stop the add-on as I wrote above. I'll fix this, but since this docs update is targeting current branch, we should write what works right now.

Reply:
Ok, this should be fixed now.